meta/config_manager/load.yaml

```diff
@@ -0,0 +1,30 @@
+---
+argument_spec:
+
+  ansible_network_os:
+    description:
+      - Set the name of the Ansible network OS platform.  This value should be
+        set to `vyos` for this provider.
+    required: true
+
+  config_manager_text:
+    description:
+      - Provide the network device configuration as a single string.  The
+        configuration text will be loaded onto the target network deivce.
+    type: str
+
+  config_manager_file:
+    description:
+      - Provide a relative or absolute path to the configuration file to be
+        loaded onto the target network device.
```

Comment:
I'm not familiar with argument_spec, but if it allows it - would be nice to repeat the constraint here that config_manager_text is mutually exclusive with config_manager_file.